src/full/Agda/TypeChecking/Monad/Base.hs

```diff
@@ -4331,7 +4332,8 @@ data TypeError
             -- ^ The given relevance does not correspond to the expected relevane.
         | UninstantiatedDotPattern A.Expr
         | ForcedConstructorNotInstantiated A.Pattern
-        | IllformedProjectionPattern A.Pattern
+        | IllformedProjectionPatternA A.Pattern
+        | IllformedProjectionPatternC C.Pattern
```

Comment:
Could we have `...A` spelled out as `...Abstract` and `...C` as `...Concrete`?
(More future proof when we should decide to print error names to the user.)

Reply:
Done!